mockup/patterns/filemanager/pattern.js

```diff
@@ -189,7 +189,8 @@ define([
           findinFilesView.triggerView
         ],
         id: 'find',
-        app: self
+        app: self,
+        disable: function() {}
```

Comment:
Disables disabling, because disabling "find" does not make sense in this context (other buttons may be disabled when the managed files are read only, but that does not prevent search).